requirements.txt

```diff
@@ -8,4 +8,4 @@ gpsoauth==0.3.0
 coveralls==1.1
 werkzeug==0.11.10
 sqlalchemy==1.0.14
--e git+https://github.com/tejado/pgoapi.git@0811db23d639039f968a82e06c7aa15a0a5016b6#egg=pgoapi
+-e git+https://github.com/keyphact/pgoapi.git#egg=pgoapi
```

Comment:
I know, but it was somehow still kinda connected to the github ^^
I got the compiling working, but now my workers are stuck @ "Exception" status and the logs tell me this:
https://i.gyazo.com/54b34c702cfb58dd9e59abef340d78cd.png

No clue what to do now, i will probably not be the only one getting all these things